extension/src/popup/components/account/AssetDetail/index.tsx

```diff
@@ -272,12 +271,16 @@ export const AssetDetail = ({
                 <img src={StellarLogo} alt="Network icon" />{" "}
                 <div>{canonical.code}</div>
               </div>
-              <div>{formatAmount(balanceTotal)}</div>
+              <div>
+                {formatAmount(total)} {canonical.code}
+              </div>
             </div>
             <div className="AssetDetail__info-modal__available-box">
               <div className="AssetDetail__info-modal__balance-row">
                 <div>{t("Total Balance")}</div>
-                <div>{formatAmount(balanceTotal)}</div>
+                <div>
+                  {formatAmount(total)} {canonical.code}
```

Comment:
any reason not to cache this is a var?  const balanceTotal = `${formatAmount(balanceTotal} {canonical.code}`

Reply:
oh yeah good call, I just replaced the references but will pull that into a var.

Reply:
fixed in [3c2fb17](https://github.com/stellar/freighter/pull/872/commits/3c2fb175d36c898ae4960a8c01d622621298c4fa)